healthtools/scrapers/nhif_inpatient_facilities.py

```diff
@@ -0,0 +1,77 @@
+from healthtools.scrapers.base_scraper import Scraper
+from healthtools.config import SITES, ES, SMALL_BATCH_NHIF
+
+
+class NhifInpatientScraper(Scraper):
+    """Scraper for the NHIF accredited inpatient facilities"""
+    def __init__(self):
+        super(NhifInpatientScraper, self).__init__()
+        self.site_url = SITES["NHIF-INPATIENT"]
+        self.fields = ["hospital", "postal_addr", "beds", "branch", "category", "id"]
+        self._type = "nhif-inpatient"
+        self.s3_key = "data/nhif_accredited_inpatient_facilities.json"
```

Comment:
@DavidLemayian not yet, I don't have access to the aws to create the files, on renaming, on it 👍